Specialisation Theory/Specialisation_Theory.txt

```diff
@@ -0,0 +1,121 @@
+const [stateObject, setStateObject] = useState ("an example of text to display in this state"). This successful introduction of Hooks has given functional components the access to Lifecycle and State in a way that has overtaken then need to use Classes in React at all if the developer so chooses.
+
+
+
+4. Design the perfect door - what should it look like, what are the components for it? What design heuristics should it follow, and how does your design match? What made you choose this design? 
+(20 marks). 
+ (a) Consider in particular (likely need to do independent learning):
+    who are your stakeholders? What are their personas? What are the doors requirements and how will your stakeholders benefit from your solution?
+
+Can you actually improve on a Norman door? It follows the concept of affordances and also the principal that the right door is one that can be used by anyone correctly first and every time without the need of any explanation. The only way I can personally see to perhaps add to it, is add a few more details for accessibility / inclusivity and post covid details for a particular placement. Please see attached design. 
+
+For the purposes of this, the stakeholders in mind would be owners of a public space (really could be anywhere with a large amount of workers/visitors) and in this case I have chosen to demonstrate with a toilet block. This gives requirements of being used frequently, potentially by busy/hurrying people by people of all abilities, with as minimal contact as possible. 
+
+The door is designed for the entry from the public space into a toilet area. - although again, could work into an office for example, equally as well - The door would have a push plate, the lack of a handle denotes what a user should do immediately, without the need for "push"(affordance), and the longer line of the plate would allow it to be at the right height/easy to push for as many people as possible. On top of this, someone with limited sight would be able to touch at arms length and be able to feel push plate.  A push is easier for someone of limited ability, and also allows for 'hands free' if someone is entering and does not wish to or is not able to touch the door. Ideally it would have a mechanical hinge to allow for controlled closing, and not a huge open swing, but to allow for users to be able to be somewhat controlled, a glass or see-through panel would help to be aware of what would be on the other side.
+
+The reverse side of the door would ideally be identical. A two way door eliminates the need for a pull, so no handle (again push plate to denote push). This becomes especially relevant on exit from a toilet block, as when people have just washed their hands they are able to exit without the need to touch the door. 
+
```

Comment:
10/20 - Well done but Design iterations of door design with details for/against iteration are not provided and 
discussion of heuristics with regards to the design